configs/setup_golden_image.py

```diff
@@ -19,10 +19,10 @@
 
 from pdb import set_trace
 
-from configs.build_config import ManifestingConfiguration
-from tmms.utils import customize_node
 from tmms.utils import file_utils
-from tmms.utils import utils as tmms_utils
+from tmms.utils import customize_node
+from tmms.utils import core_utils as tmms_utils
```

Comment:
Okay, this is a problem.   Up until now I didn't understand the need for core_utils because it just looks like a new namespace.  Fine, I'll play along until right here.  This is a semantic namespace violation: you stopped using tmms.utils, went to core_utils, just to rename it "as tmms_utils".   This is too confusing for long term support.

Right now there's no difference between core and uncore utils.

Reply:
> Right now there's no difference between core and uncore utils.

"Imports" is the major difference. core_utils.py using ONLY standard libraries that comes with python. utils.py require "apt install". If you try to install manifesting from source on the absolutely fresh system - it will fail, because utils.py dependencies are not met.

Reply:
git clone manifesting
cd manfiesting
./setup

Reply:
So you found things that weren't needed during this operation of setup.py that had unresolved dependencies, and that's what's different between utils and core_utils?

Reply:
Yes.
setup_environment.py would "import utils" to use "piper", but utils had "psutil" and "requests" imports that requires install of python3-psutil and python3-requests...

Reply:
Why not just add them to packages.req and install them alongside other Python debs?   This seems like a lot of extra work for an already-solved problem.

Reply:
packages.req is not used anywhere by anything. I just added it for future dockerization process.

> and install them alongside other Python debs?

Sure. We are doing that in setup_environment 10 lines below the import of utils that requires install of dependencies...

Reply:
Not used?  It's used in install_base_packages().  It reads the file.  It runs apt-get install on each line.

Reply:
Oh yeah, I forgot I actually implemented that in this merge request... still. This doesnt change the the problem.